src/community/script/core/src/main/java/org/geoserver/script/rest/service/AppService.java

```diff
@@ -0,0 +1,107 @@
+/** (c) 2014 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2013 OpenPlans
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ *
+ * @author David Vick, Boundless 2017
+ **/
+package org.geoserver.script.rest.service;
+
+import net.sf.json.JSONArray;
+import net.sf.json.JSONObject;
+import org.geoserver.platform.resource.Resource;
+import org.geoserver.platform.resource.Resources;
+import org.geoserver.rest.RestException;
+import org.geoserver.script.ScriptManager;
+import org.geoserver.script.app.AppHook;
+import org.geotools.util.logging.Logging;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import javax.script.ScriptEngine;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static java.lang.String.format;
+
+@Service
+public class AppService {
+    static Logger LOGGER = Logging.getLogger(AppService.class);
+
+    @Autowired
+    ScriptManager scriptManager;
+
+    public ResponseEntity<?> getAppList() {
+        org.geoserver.platform.resource.Resource appRoot = scriptManager.app();
```

Comment:
I don't think you need the full package anymore, since we aren't importing the Restlet Resource anymore.

Reply:
fixed.